add missing include to fix compilation error with GCC 12.x
../../3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp:300:49: error: no matching function for call to begin(const char* const [140])
cfr. https://github.com/archlinux/svntogit-packages/blob/packages/qt5-webengine/trunk/qt5-webengine-gcc12.patch
--- Qt5/5.15.7/GCCcore-12.2.0/qt-everywhere-src-5.15.7/qtwebengine/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp.orig	2021-10-19 14:46:32.000000000 +0200
+++ Qt5/5.15.7/GCCcore-12.2.0/qt-everywhere-src-5.15.7/qtwebengine/src/3rdparty/chromium/third_party/skia/src/utils/SkParseColor.cpp	2022-11-25 14:48:38.501288572 +0100
@@ -8,6 +8,8 @@
 
 #include "include/utils/SkParse.h"
 
+#include <iterator>
+
 static constexpr const char* gColorNames[] = {
     "aliceblue",
     "antiquewhite",
